### PR TITLE
Quick release panel clips

### PR DESCRIPTION
**Pull request by ScramblerUSA**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Quick Release Panel Clips
image

#### How was it tested? 
Printed and fitted on my Voron 2.4
Judging by STL files this mod must work on 1.8/Trident as well as they use identical stock clips.

#### Any background context you want to provide?
Heavily inspired by Annex Engineering's Denali (https://github.com/Annex-Engineering/Denali) panel clips.
image
I liked the idea but found them to be too bulky and not fitting to Voron design. Hence I created my own.

### CODE REVIEW

File: printer_mods/ScramblerUSA/QuickReleaseClips/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Voron-style panel clips inspired by Annex-Engineering Denali printer's clips (https://github.com/Annex-Engineering/Denali/tree/master/STLs/Panel_Mounting/Side_Top_and_Rear_Panel_Quick_Release)
+
+![](./images/clips.jpg)
+
+I found Denali's clips too bulky and not really fitting to the overall style of Voron printer, so I designed these.
+
+## Parts Required
+
+None! You can even free up a few M8 screws and hammerhead nuts you are currently using.
```
Comment (MapleLeafMakers) [inline printer_mods/ScramblerUSA/QuickReleaseClips/README.md:11]: M3x8 screws, not M8 screws.

Comment (ScramblerUSA) [reply printer_mods/ScramblerUSA/QuickReleaseClips/README.md]: D'oh! That's exactly what I meant to say.

**Comment by MapleLeafMakers** [comment]
LGTM

**Comment by cbc02009** [comment]
As much as I love this mod, I don't think the Annex license is compatible with GPL v3 (which the usermods repo uses) . I don't think we can add this one, but let me double check with the team.

**Comment by ScramblerUSA** [comment]
@cbc02009 I am not a lawyer, but they actually have a human-friendly license digest right here: https://github.com/Annex-Engineering/ANNEX-Engineering-License-Agreement/blob/main/LICENSE.md and the last item says:
```
- You want to use our stuff to make your own original things (that are not based on Annex-files) and sell them?
- Great. Go for it, we love this.
```
This mod is based on the IDEA, not the FILES - I created the entire thing from scratch (or should I say from sketch) in Fusion 360.  I think it qualifies minus the fact I'm not selling it. This is basically a clone, not a derivative. This clearly would violate a patent if there was one.

**Comment by FHeilmann** [comment]
We're not lawyers either. So we'd like to cover our bases, even if it ends up being excessive. If you'd like us to accept this mod into this repository, please obtain permission from the annex community or the member of the annex Community this mod takes the idea from. I'm afraid this is going to be nonnegotiable.

**Pull request closed without merging**